dev/ProjectReunion_DLL/COMExports.cpp

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "EncodedLaunchExecuteCommand.h"
+#include "Association.h"
+
+using namespace winrt::Microsoft::ProjectReunion::implementation;
+
+std::wstring GetDllModulePath();
+std::wstring GetCoClassPath(const GUID& iid);
+void RegisterCoClass(const GUID& iid);
+void UnregisterCoClass(const GUID& iid);
```

Comment:
Why?

Is this pre-DynamicDependencies temporary workaround or something more lasting?

Reply:
You're going to have to articulate your question differently.

Reply:
Let me rephase: Do you intend these functions to exist long-term and if so, why? Or is this a short-term workaround to enable your code to run and test until the Dynamic Depenendencies PR's checked in?

I can see why they're needed before DynDep. I don't see any comment about this being a temporary workaround for DynDep's absence and I'm not aware of a reason they'd be needed longer-term. So I was wondering if I'm missing something.

Reply:
I think we went over this in another comment.  Yes, this is temporary until we have a different solution.